packages/flutter/test/material/stepper_test.dart

```diff
@@ -846,4 +846,65 @@ void main() {
       expect(listView.physics, physics);
     }
   });
+
+  testWidgets('Stepper custom elevation', (WidgetTester tester) async {
+    const double elevation = 4.0;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Container(
+            width: 200,
+            height: 75,
+            child: Stepper(
+              type: StepperType.horizontal,
+              horizontalElevation: elevation,
+              steps: const <Step>[
+                Step(
+                  title: Text('Regular title'),
+                  subtitle: Text('Text subtitle'),
+                  content: Text('Text content'),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+
+    await tester.pump();
```

Comment:
You don't require `tester.pump()` here. It's usually used to rebuild the same widget again as we do for `StatefulWidget` or during animations.

You can remove it, and it will still work fine.

Reply:
This is the final nit I can find in the PR

Someone from team will have a closer look, and guide you if any changes are needed.

Reply:
Same for another test too. You can remove the `tester.pump`.

Reply:
Done